js/menu.js

```diff
@@ -69,9 +69,14 @@ function init() {
     });
   });
 
-  $('[data-fl-toggle-menu]').click(function (event) {
-    event.preventDefault();
-    $('.fl-viewport-header .hamburger').toggleClass('is-active');
-    $('body').toggleClass('has-slide-menu');
+  $('[data-fl-toggle-menu]').on('click keydown', function(event) {
+    if (event.type === 'click' || event.which === 32 || event.which === 13) {
```

Comment:
what's 32?

Reply:
don't indent the whole code, just do an early return:

```
if (event.type === 'keydown' && is not return key) {
  return
}

// rest of logic here
```

Reply:
32 it is spacebar key

Reply:
Regarding early return. We have to treat both click and keydown event and click event. And treat a little bit differently. Therefore I think we cannot do an early return.

Reply:
I still don't understand why an early return is not possible.

```
if (event.type === 'keydown' && is not return key) {
  return
}

// rest of logic here

if (event.type === 'whatever) {
  // 1
}

// 2
```

If you read through our code styles you'll find that you should return as early as possible instead of indenting code into blocks whenever possible.

Reply:
Fixed.